core/src/main/java/org/infinispan/cache/impl/SimpleCacheImpl.java

```diff
@@ -92,14 +96,12 @@
 import org.infinispan.util.TimeService;
 import org.infinispan.util.concurrent.CompletableFutures;
 import org.infinispan.util.concurrent.locks.LockManager;
-import org.infinispan.util.function.SerializableBiConsumer;
 import org.infinispan.util.logging.Log;
 import org.infinispan.util.logging.LogFactory;
 
 /**
  * Simple local cache without interceptor stack.
  * The cache still implements {@link AdvancedCache} since it is too troublesome to omit that.
- *
```

Comment:
I see you are removing the empty line between the javadoc and the annotations in many places. why????